lib/config/presets/internal/replacements.ts

```diff
@@ -698,6 +699,23 @@ const mui: PresetTemplate = {
   title: 'material-ui-to-mui',
 };
 
+const k8sImages: string[] = JSON.parse(dataFiles.get('data/k8s-images.json')!);
```

Comment:
```suggestion
const k8sImages = JSON.parse(dataFiles.get('data/k8s-images.json')!) as string[];
```
@JamieMagee isn't that better for our future type changes?

Reply:
It doesn't matter either way. I'll have to define a zod schema for it later.

Reply:
Let's keep it as-is for consistency with the other usages?

Reply:
maybe add zod schema now, so we don't have to do it later. a string array should be pretty easy.

Reply:
Added zod parsing, was indeed not too difficult :)